packages/language/src/document.js

```diff
@@ -634,11 +704,162 @@ Document.prototype.detectSentiment = function(options, callback) {
+ *
+ * @param {object=} options - Configuration object. See
+ *     [documents.annotateText](https://cloud.google.com/natural-language/reference/rest/v1/documents/analyzeSyntax#request-body).
+ * @param {boolean} options.verbose - Enable verbose mode for more detailed
+ *     results. Default: `false`
+ * @param {function} callback - The callback function.
+ * @param {?error} callback.err - An error occurred while making this request.
+ * @param {object} callback.syntax - The recognizd syntax from the text.
+ * @param {string[]} callback.syntax.sentences - Sentences detected from the
+ *     text.
+ * @param {object[]} callback.syntax.tokens - Parts of speech that were
+ *     detected from the text.
+ * @param {string} callback.syntax.language - The language detected from the
+ *     text.
+ * @param {object} callback.apiResponse - The full API response.
+ *
+ * @example
+ * document.detectSyntax(function(err, syntax) {
+ *   if (err) {
+ *     // Error handling omitted.
+ *   }
+ *
+ *   // syntax = {
+ *   //   sentences: [
+ *   //     'Google is an American multinational technology company ' +
+ *   //     'specializing in Internet-related services and products.'
+ *   //   ],
+ *   //   tokens: [
+ *   //     {
+ *   //       text: 'Google',
+ *   //       partOfSpeech: 'Noun (common and proper)'
+ *   //     },
+ *   //     {
+ *   //       text: 'is',
+ *   //       partOfSpeech: 'Verb (all tenses and modes)'
+ *   //     },
+ *   //     ...
+ *   //   ],
+ *   //   language: 'en'
+ *   // }
+ * });
+ *
+ * //-
+ * // Verbose mode may also be enabled for more detailed results.
+ * //-
+ * var options = {
+ *   verbose: true
+ * };
+ *
+ * document.detectSyntax(options, function(err, syntax) {
+ *   if (err) {
+ *     // Error handling omitted.
+ *   }
+ *
+ *   // syntax = {
+ *   //   sentences: [
+ *   //     {
+ *   //       text: {
+ *   //         content:
+ *   //           'Google is an American multinational technology company' +
+ *   //           'specializing in Internet-related services and products.',
+ *   //         beginOffset: -1
+ *   //       },
+ *   //       sentiment: {
+ *   //         score: 100
+ *   //         magnitude: 4
+ *   //       }
+ *   //     }
+ *   //   ],
+ *   //   tokens: [
+ *   //     {
+ *   //       text: {
+ *   //         content: 'Google',
+ *   //         beginOffset: -1
+ *   //       },
+ *   //       partOfSpeech: {
+ *   //         tag: 'NOUN',
+ *   //         aspect: 'PERFECTIVE',
+ *   //         case: 'ADVERBIAL',
+ *   //         form: 'ADNOMIAL',
+ *   //         gender: 'FEMININE',
+ *   //         mood: 'IMPERATIVE',
+ *   //         number: 'SINGULAR',
+ *   //         person: 'FIRST',
+ *   //         proper: 'PROPER',
+ *   //         reciprocity: 'RECIPROCAL',
+ *   //         tense: 'PAST',
+ *   //         voice: 'PASSIVE'
+ *   //       },
+ *   //       dependencyEdge: {
+ *   //         headTokenIndex: 1,
+ *   //         label: 'NSUBJ'
```

Comment:
This would be similar to partsOfSpeech. We can return both-- the code is good for computers, the friendly name is good for humans; an example use would be for logging/printing to a user.